components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -454,12 +453,32 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
         try {
             String[] userNames = null;
             if (!SCIMConstants.UserSchemaConstants.GROUP_URI.equals(attributeName)) {
+
+                if (!filterOperation.equalsIgnoreCase(SCIMCommonConstants.EQ)) {
```

Comment:
Usually constant must be in other side

Reply:
+1. Here filterOperation cannot be null.